docs/section5/Rule5-39.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-39  
+
+**Rule ID:** 5-39  
+**Rule Description:** Automatically controlled dynamic glazing may be modeled. Manually controlled dynamic glazing shall use the average of the minimum and maximum SHGC and VT.  
+**Rule Assertion:** P-RMR subsurface: dynamic_glazing_type = expected value  
+**Appendix G Section:** Section G3.1-5(a)5 Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** Yes  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:  
+
+- For each zone in the Proposed model: `for zone_p in P_RMR...zones:`
+
+  - For each surface in zone: `for surface_p in zone_p.surfaces:`
+
+    - For each subsurface in surface: `for subsurface_p in surface_p.subsurfaces:`
+
+      - Check if subsurface has dynamic glazing, set rule applicability check to True: `if subsurface_p.dynamic_glazing_type != "NOT_DYNAMIC: rule_applicability_check = TRUE`
+
+        **Rule Assertion:**  
+
```

Comment:
Since we are expanding this RDS to ialso check that dynamic glazing is of the same type in P and U RMR, I would make Case 1 "if subsurface_p.dynamic_glazing_type !=subsurface_U.dynamic_glazing_type: FAIL and show_message "Dynamic glazing type does not match in the proposed design versus user model."